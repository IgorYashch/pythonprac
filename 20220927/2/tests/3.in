12,41,78,21,44,51,3768,236,3455,0,34,13,2
