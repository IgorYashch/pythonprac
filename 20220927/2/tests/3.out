[0, 51, 2, 3768, 3455, 44, 21, 12, 34, 13, 41, 78, 236]
